src/backends/dummy.rs

```diff
@@ -37,12 +37,19 @@
 
 use Tracer;
 use errors::HWTracerError;
-use HWTrace;
-use std::ptr;
+use Trace;
 #[cfg(debug_assertions)]
 use std::ops::Drop;
 use TracerState;
 
+/// An imaginary trace.
```

Comment:
I don't think this comment is entirely helpful: it's really not an imaginary trace. It's a trace that can only ever be empty, right?

Reply:
Yes, an entirely vacuous container, only existing to satisfy a type constraint really.

Reply:
Them's some big words! How about just "A trace which can only ever be empty" or similar? My brain does better with short words ;)